Source/Core/VideoBackends/Software/Tev.cpp

```diff
@@ -544,19 +544,6 @@ void Tev::Draw()
   if (!TevAlphaTest(output[ALP_C]))
     return;
 
-  // Hardware testing indicates that an alpha of 1 can pass an alpha test,
```

Comment:
I see this blurb was written by you.  You said it was hardware tested but then you have the TODO.  Does that mean there was some issue with the hardware test you performed or was there another reason you doubted the correctness of this behavior?

I see in the original PR that you mention that the second commit (the one that contains this change) you were less certain on...

Reply:
I think I see what you are saying, it was only tested in one scenario, so while there was a single case that was tested, not every case was tested and therefore it needed more investigation.

Reply:
Ok, sorry for the comment spam.  I was a bit confused initially why this was added (especially if the original logic in Software Renderer was working) but I think I fully understand the history now.  In addition to the one scenario bit above, you were also trying to resolve an issue with the hardware renders.  So in order to be consistent, you _also_ added the workaround to the software renderer.

If the plan is to fix the hardware renderers and avoid this hack altogether, then I have no concerns.  Even if not, software being the ideal solution is fine too.

Reply:
Yes, that's correct. I was trying to remove a previous issue that affected the hardware renderers, and removed an older hack (#4514) that existed in both the software renderer and the hardware renderers. I then added an alternative hack on alpha=1 that was needed to avoid regressions in fortune street caused by removing the original hack, and I added that to both the software renderer and hardware renderers.

I did investigate what happens both when the first hack is removed and the second added in https://github.com/dolphin-emu/dolphin/pull/10394#issuecomment-1024704413, and I even noted that there was "[No diff](https://fifo.ci/compare/7310048-7309670/)" for fortune-street-white-box on the software renderer, but I didn't realize at the time that that meant that the software renderer didn't regress from removing the hack and thus it didn't need the new alpha=1 hack at all. This PR removes that hack.

Incidentally, the old hack wasn't added to the software renderer in #4514, but instead was accidentally added in 1dead05cae47aeb9cb9e14a65a286e0f0de8dc95 for both color and alpha, and then fixed for color (but not alpha) in 5e1c6aba331d2a3cb0f0ce32cce78b4b9b12fa01. So, instead of being deliberately introduced there, the exact same behavior existed by accident, and thus was there for me to remove in my PR.

I'd like to remove the hack in the hardware renderers too, but I'm not sure how feasible that will be.